app/src/main/java/edu/csumb/caitlin/lo/cst438_s21_proj01_group10/MyPlayerActivity.java

```diff
@@ -62,7 +60,7 @@ private void getDB() {
      * Connect display elements to variables
      */
     private void connectToDisplay() {
-        myPlayers = findViewById(R.id.my_players_view);
+        myPlayers = findViewById(R.id.myTeamTextView);
```

Comment:
when i refactor the ID for the textview in myTeam or myPlayer, it changes both ID's in section. prob because i copy pasted keiti's myplayer xml to my myteam xml. code still all functions but just a heads up on this change.